src/main/java/org/mlflow/tracking/RiMlflowClient.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2021 Rikai Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mlflow.tracking;
+import org.mlflow.tracking.creds.*;
+
+import java.net.URI;
+
+
+/**
+ * Client to an MLflow Tracking Sever.
+ * Originally from mlflow client 1.21.0
+ */
+public class RiMlflowClient extends MlflowClient {
```

Comment:
What does `Ri` mean, `Rikai`?  If so lets just name it as `RikaiMlflowClient` and make it a package visible class?  This is not design as user-facing API right? 

Should we actually provide a patch to `mlflow` java SDK as well?

Reply:
Renamed it to `RikaiMlflowClient`, it's actually a patch of `MlflowClient`, so I keep the same visible level and package name with `RikaiMlflowClient`.
It's a good idea to provide a patch to the mlflow official repo, I will try it. And we can remove this file after patch merged by mlflow official.